### PR TITLE
Support swift 5

### PR DESCRIPTION
**Pull request by danielrobleM**
This change fix issue introduced in [PR-15](https://github.com/etoledom/Heyou/pull/15) , error on `pod spec lint`. 

Error description:
`swift_versions` (`5.0`) compared to the one present in your `.swift-version` file (`4.2`).

### CODE REVIEW

**Comment by danielrobleM** [comment]
@etoledom  not sure about to change version to 0.4.1 🤔. Let me know

**Review by etoledom — COMMENTED** [review]
As mentioned in the `pod spec lint` error:
> Please remove the `.swift-version` file which is now deprecated and only use the `swift_versions` attribute within your podspec.

Sounds better approach to just remove this file.
Could you please update with this change?

Thank you! 🙏

**Comment by etoledom** [comment]
> @etoledom not sure about to change version to 0.4.1 🤔. Let me know

I like that idea! We can do this as a hot-fix. Let's do it :)

File: Example/Pods/Pods.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
```
Comment (danielrobleM) [inline Example/Pods/Pods.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist:2]: @etoledom I am using xcode 11.3 (11C29).  
Not sure why these get generate now (https://stackoverflow.com/questions/49564513/new-file-created-in-xcode-9-3-wsname-xcworkspace-xcshareddata-ideworkspaceche)

**Comment by danielrobleM** [comment]
Move to https://github.com/etoledom/Heyou/pull/17

**Pull request closed without merging**